### PR TITLE
Patch For Mobile View

### PR DESCRIPTION
**Pull request by deb-cod**
### **User description**
Sizing all the meter boxes to same.

___

### **PR Type**
Enhancement

___

### **Description**
- Added padding-bottom to the main container for better spacing.
- Adjusted width and padding for `testArea` and `testArea2` to ensure consistent sizing.
- Unified styles for `testArea` and `testArea2` to reduce redundancy.
- Updated media queries to enhance responsiveness for different screen sizes and heights.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>index.html</strong><dd><code>Improve mobile view styling and responsiveness</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

index.html

<li>Added padding-bottom to container.<br> <li> Adjusted width and padding for test areas.<br> <li> Unified styles for <code>testArea</code> and <code>testArea2</code>.<br> <li> Updated media queries for responsive design.<br>

</details>

  </td>
  <td><a href="https://github.com/talview/speedtest/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051">+29/-35</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: docker/standalone.php [hunk 1]
```diff
@@ -394,26 +395,15 @@ function initUI() {
...
+			width: 100%;
 
 		}
 
-		div.testArea2 {
+		div.testArea, div.testArea2 {
 			display: flex;
-			width: 248px;
+			width:100%;
+			width: 250px;
```
Comment (harish-talview) [inline docker/standalone.php]: you have two width. Fix this.

File: docker/standalone.php [hunk 2]
```diff
@@ -502,24 +493,27 @@ function initUI() {
...
-                gap: 6px !important;
-            }
-
-            .title-class {
-                font-size: 24px !important;
-            }
+        @media (max-width: 750px) {
+			.container .testArea,
+			.container .testArea2 {
+				max-width: 150px;
```
Comment (harish-talview) [inline docker/standalone.php]: We discussed this. No max-width for mobile

File: docker/standalone.php (hunk 1 above)
Comment (deb-cod) [reply docker/standalone.php]: max-width: 250px; is required I am fixing this.

File: docker/standalone.php (hunk 2 above)
Comment (deb-cod) [reply docker/standalone.php]: fixing this right away

**Review by harish-talview — APPROVED** [review]

**Pull request merged**